src/test/regress/sql/multi_table_ddl.sql

```diff
@@ -75,17 +77,39 @@ ALTER SEQUENCE testserialtable_id_seq OWNED BY NONE;
 -- or create a sequence with a distributed owner
 CREATE SEQUENCE standalone_sequence OWNED BY testserialtable.group_id;
 
--- or even change a manual sequence to be owned by a distributed table
-CREATE SEQUENCE standalone_sequence;
-ALTER SEQUENCE standalone_sequence OWNED BY testserialtable.group_id;
```

Comment:
This `owned_by` operation is used when adding a new column with a serial prototype. For this reason I removed the check from the code, and also its test. So, currently this can be done, but I figured supporting serial prototypes is more important right now.

To error out for this operation with a manual sequence, and not error out for serial prototypes, `ALTER SEQUENCE` command needs to be handled in this special case, so we can keep a note of this and fix it in the possible future `ALTER SEQUENCE` PR.